ocdata/vasp.py

```diff
@@ -31,7 +31,7 @@
               'pp': 'PBE',
               'xc': 'PBE'}
 
-VASP_PP_PATH = '/private/home/sidgoyal/vasp_pp/5.4'
+VASP_PP_PATH = '/private/home/sidgoyal/vasp_pp/'
```

Comment:
could you rebase from master as well?
It seems there's a merge conflict if were to merge to master.